### PR TITLE
minor mistakes

### PR DESCRIPTION
**Pull request by MrKrishnaAgarwal**

### CODE REVIEW

**Review by jonmeow — CHANGES_REQUESTED** [review]
Preexisting and preprocessor are both the preferred spelling. Hyphenating is incorrect.

e.g.: https://en.cppreference.com/w/cpp/preprocessor

**Comment by MrKrishnaAgarwal** [comment]
> Preexisting and preprocessor are both the preferred spelling. Hyphenating is incorrect.
> 
> e.g.: https://en.cppreference.com/w/cpp/preprocessor

Sure
I'll make the changes

File: docs/project/evolution.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ a pull request instead go through the full evolution process.
 ### Proposal PRs
 
 A proposal PR should use the `proposal` label, have a descriptive title, and
-easily understood initial summary comment. Authors and leads are encouraged to
+easily understand initial summary comment. Authors and leads are encouraged to
```
Comment (jonmeow) [inline docs/project/evolution.md]: "understood" is correct here. As usual, please undo this change.

**Review by jonmeow — APPROVED** [review]

**Comment by MrKrishnaAgarwal** [comment]
Thanks 👍🏻

**Pull request merged**